scene/resources/theme.cpp

```diff
@@ -1605,6 +1642,7 @@ void Theme::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("merge_with", "other"), &Theme::merge_with);
 	ClassDB::bind_method(D_METHOD("clear"), &Theme::clear);
 
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "default_base_scale"), "set_default_base_scale", "get_default_base_scale");
```

Comment:
Maybe add a property hint for easy edition? E.g. "0.01,4.0,0.01,or_greater" maybe?
That's going a bit low in the sub-1.0 range but with a step of 0.01 I don't think it makes much sense to impose an arbitrary limit on e.g. "0.25", or "0.1".

Reply:
I'd cap the upper limit at `2.0`. That's the value we use for HiDPI and that's the maximum provided value for the editor scale, 200%. `or_greater` will handle the rest nicely.

Reply:
That's fine until 8K becomes mainstream :P